app/resources/api/v2/specific_tube_rack_creation_resource.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
```

Comment:
It feels like we shouldn't be adding new resources with todos in them?

Reply:
removed and comments added